src/main/java/hudson/views/UserRelevanceFilter.java

```diff
@@ -14,16 +14,18 @@
 
 import java.lang.reflect.Method;
 import java.util.List;
+import java.util.Locale;
 
 import org.kohsuke.stapler.DataBoundConstructor;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
 /**
  * Filters jobs that are relevant to the user.
  * @author Jacob.Robertson
  */
 public class UserRelevanceFilter extends AbstractBuildTrendFilter {
 
-	private static final String ANONYMOUS = Hudson.ANONYMOUS.getName().toUpperCase();
+	private static final String ANONYMOUS = Hudson.ANONYMOUS.getName().toUpperCase(Locale.getDefault());

```

Comment:
I think you meant
```suggestion
	private static final String ANONYMOUS = Hudson.ANONYMOUS.getName().toUpperCase(Locale.ROOT);

```
since we do _not_ want to use the system locale for this (though in the absence of the letter `i` it is unlikely to matter).